### PR TITLE
Lavinia

### PR DESCRIPTION
**Pull request by IntakeValori**

### CODE REVIEW

File: src/main/java/pages/ContactUsPage.java [hunk 1]
```diff
@@ -12,6 +12,14 @@
     private int timeOut = 5;
 
     private By contactUsPageTitle = By.xpath("//h1[contains(text(),'Contact')]");
+    private By firstName = By.id("form-input-firstName");
+    private By lastName = By.id("form-input-lastName");
+    private By companyName = By.id ("form-input-companyName");
+    private By phone = By.id("form-input-homePhone");
+    private By email = By.id("form-input-email");
+    private By message = By.id("form-input-message");
+    private By sendButton = By.xpath("/html/body/div[2]/main/div/section/article/div[2]/form/fieldset/div[5]/button");
```
Comment (IntakeValori) [inline src/main/java/pages/ContactUsPage.java:21]: This xpath is as fragile as they come. It starts from the bottom of the page and directs all the way to the button.

This definitely needs to change to a more robust locator.

File: src/main/java/pages/ContactUsPage.java [hunk 2]
```diff
@@ -24,4 +32,45 @@ public Boolean isTitleVisible(){
...
+
+    public Boolean isMessageVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(message));
+        return driver.findElement(message).isDisplayed();
+    }
+    public Boolean isSendVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(sendButton));
+        return driver.findElement(sendButton).isDisplayed();
```
Comment (IntakeValori) [inline src/main/java/pages/ContactUsPage.java:73]: Lots of code duplication in the above six methods. Can you find a good way to reduce the duplication and have the code abide the SOLID principles better?

File: src/main/java/pages/ContactUsPage.java (hunk 1 above)
Comment (laviniamicu78) [reply src/main/java/pages/ContactUsPage.java:21]: What about this ? 
private By sendButton = By.xpath("//button[text()=\"Versturen\"]");
Is better than this ?
private By sendButton = By.xpath("/html/body/div[2]/main/div/section/article/div[2]/form/fieldset/div[5]/button");

Comment (IntakeValori) [reply src/main/java/pages/ContactUsPage.java:21]: Because the website is bilingual you should refrain from using the text to build your locators.

Comment (laviniamicu78) [reply src/main/java/pages/ContactUsPage.java:21]: Shall we talk about your comments and resolve the conversation ?  In the first week of April?

**Pull request closed without merging**